webview-ui/test-generation/src/state.ts

```diff
@@ -360,6 +365,26 @@ export const reducer = (current: State, action: Action): State => {
         },
       };
     }
+    case 'loadAppNames': {
+      let apps = action.value;
+      apps = apps.filter(
+        (item) =>
+          item.metadata === null ||
```

Comment:
I'm a wee bit confused, since according to the `AppInfo` interface, metadata is always defined. Which is it?

Reply:
I'll remove that, I think the metadata can be null, but very rarely, so not sure if worth adding to the `AppInfo` interface.

Reply:
I'd vote for adding it to the interface. The downside is a few extra null checks, upside is an accurate and reliable type.